mvlearn/embed/kcca.py

```diff
@@ -111,53 +167,48 @@ def fit(self, Xs):
         """
```

Comment:
Fit should return self, not weights_. See comment below about return from transform().

Changing this will mean you also need to change the tutorial notebook

Reply:
just fixed